src/main/java/org/isf/utils/image/ImageUtil.java

```diff
@@ -33,11 +34,12 @@
     private ImageUtil() {
     }
 
-    public static BufferedImage scaleImage(final BufferedImage src,
+    public static BufferedImage scaleImage(final Image src,
                                            final int boundWidth,
                                            final int boundHeight){
-        final int originalWidth = src.getWidth();
-        final int originalHeight = src.getHeight();
+    	BufferedImage bi = toBufferedImage(src);
+        final int originalWidth = bi.getWidth();
+        final int originalHeight = bi.getHeight();
```

Comment:
Inconsistent indentation (line 40 vs. lines 41 and 42).

Reply:
I corrected as you suggested.